envConfig/local.env

```
@@ -3,7 +3,7 @@ SIMORGH_BASE_URL=http://localhost:7080
 SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN=http://localhost:7080
 SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/
 SIMORGH_ATI_BASE_URL=https://logws1363.ati-host.net?
-SIMORGH_TOGGLES_URL=https://toggles.test.api.bbci.co.uk
+SIMORGH_TOGGLES_URL=https://config.test.api.bbci.co.uk/
```

It'd be good to also update this environment variable name throughout the application, to reflect that this is now a Config endpoint, not just one for toggles.
```suggestion
SIMORGH_CONFIG_URL=https://config.test.api.bbci.co.uk/
```